.github/workflows/main.yml

```diff
@@ -1 +1,20 @@
 name: CI
+
+on:
+  pull_request:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  test:
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@v1
+    with:
+      envs: |
+        - linux: py311-astropy50
+        - macos: py311-astropy51
+        - windows: py311-astropy52
+        - linux: py311-astropy53
+        - macos: py311-astropy60
```

Comment:
This might be way overkill.

Really, we only run the PR benchmark on `main` and the latest backport branch, and only on Linux (https://github.com/astropy/astropy/blob/main/.github/workflows/ci_benchmark.yml). For the rest, if `asv` fails to run a benchmark, it simply skips over to the next commit, so I don't see why everything has to run for all the older versions.

```suggestion
```

Reply:
@pllim - this is not for the CI but in case we ever want to produce the website with graphs etc. There have been issues in the past where if we added an import at the top of the coordinates benchmark, then if that benchmark didn't exist in an old version of astropy you wouldn't be able to run *any* of the benchmarks for coordinates on that older astropy version. So it's a good idea to make sure the benchmarks run on a variety of astropy versions.

Reply:
(The total CI time here is also pretty small and PRs to this repo are rare so I don't see the downside of having a few configs)